Uplift/Views/SportsDetail/SportsDetailDiscussionCollectionViewCell.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SportsDetailDiscussionCollectionViewCell.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 4/18/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SportsDetailDiscussionCollectionViewCell: UICollectionViewCell {
+    
+    private let headerLabel = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func setupViews() {
+        headerLabel.text = "DISCUSSION"
```

Comment:
Can we add "DISCUSSION" as a constant to `ClientStrings.swift` and use that value instead? That's where we keep all of the client-facing strings.